installer/bundle/bundle_skel.sh

```diff
@@ -370,6 +383,11 @@ EOF
     $tempFile 1> /dev/null 2> /dev/null
     [ $? -eq 0 ] && hasCtypes=0
     rm $tempFile
+
+    # Check #2: Attempt to run python with the single import command
+    python -c "import ctypes" 1> /dev/null 2> /dev/null
```

Comment:
I noticed in the code that both forms are used; I may standardize this in a later review.